MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -389,9 +389,10 @@ private static TranscodeReason GetTranscodeReasonForFailedCondition(ProfileCondi
         /// <param name="type">The <see cref="DlnaProfileType"/>.</param>
         /// <param name="playProfile">The <see cref="DirectPlayProfile"/> object to get the video stream from.</param>
         /// <returns>The normalized input container.</returns>
-        public static string NormalizeMediaSourceFormatIntoSingleContainer(string inputContainer, DeviceProfile? profile, DlnaProfileType type, DirectPlayProfile? playProfile = null)
+        public static string? NormalizeMediaSourceFormatIntoSingleContainer(string inputContainer, DeviceProfile? profile, DlnaProfileType type, DirectPlayProfile? playProfile = null)
         {
-            if (profile is null || !inputContainer.Contains(',', StringComparison.OrdinalIgnoreCase))
+            // If the source is Live TV the inputContainer will be null until the mediasource is probed on first access
+            if (profile is null || inputContainer is null || !inputContainer.Contains(',', StringComparison.OrdinalIgnoreCase))
```

Comment:
Use `string.IsNullOrEmpty` to check `inputContainer` for weird LiveTV plugins that may supply empty string.